chrome-extension/src/index.js

```diff
@@ -124,3 +119,9 @@ if (!window.location.ancestorOrigins.contains(extensionOrigin) && !document.getE
     }
   });
 }
+
+window.addEventListener('click', e => {
+  if (document.querySelector('._1fng') && !isHaveBtnAleady()) {
```

Comment:
@ysm0622 `._1fng` does not look like a meaningful name. Everything else is good 👍

Reply:
`_1fng` is FB's specific component hashed class name.